terraform/groups/ewf/locals.tf

```diff
@@ -10,5 +10,6 @@ locals {
   ingress_cidr_blocks = var.internal_access_only ? concat(values(data.vault_generic_secret.internal_cidrs.data), var.internal_access_cidrs, local.test_cidr_blocks) : concat(jsondecode(data.vault_generic_secret.external_cidrs.data["external_cidrs"]), var.internal_access_cidrs)
   subnet_ids          = var.internal_access_only ? data.aws_subnet_ids.data_subnets.ids : data.aws_subnet_ids.public_subnets.ids
 
-
+  iboss_cidr_blocks = jsondecode(data.vault_generic_secret.iboss_cidrs.data_json)
+  iboss_cidrs       = tolist(local.iboss_cidr_blocks["iboss_cidrs"])
```

Comment:
The `tolist` function is unnecessary here as the data is already stored as a list within Vault.